lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/MembershipInvtnSignedToken.json

```diff
@@ -10,6 +10,11 @@
     "membershipInvitationId": {
       "type": "string",
       "description": "The ID of the membership invitation."
+    },
```

Comment:
as we discussed, this is not needed, being redundant w.r.t. the expiration in the invitation